src/formatDate.js

```diff
@@ -1,14 +1,44 @@
 'use strict';
 
-/**
+/*
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateObj = {};
+  const delimiter = fromFormat.find((f) => f.length === 1);
```

Comment:
The assumption that the delimiter is a single character in the `fromFormat` array might not always hold true. Consider scenarios where the delimiter is not present or is more than one character.